### PR TITLE
Release Notes for 1.42.0.Final Version

### PR DESCRIPTION
**Pull request by ricardozanini**
RN for 1.42.0.

- [x] - Add issues from repos

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -3,9 +3,25 @@
...
+* link:https://issues.redhat.com/browse/KOGITO-9525[KOGITO-9525] - Swagger documentation is not properly generated if description is null
+* link:https://issues.redhat.com/browse/KOGITO-9434[KOGITO-9434] - Python modules should not be compiled by default
+* link:https://issues.redhat.com/browse/KOGITO-9520[KOGITO-9520] - Updating Java SDK to 4.0.4
+* link:https://issues.redhat.com/browse/KOGITO-9501[KOGITO-9501] - Changing Everit to json-schema-validator
+* link:https://issues.redhat.com/browse/KOGITO-9480[KOGITO-9480] - Bump quarkus-reactive-messaging-http version to 1.1.5
+* link:https://issues.redhat.com/browse/KOGITO-9526[KOGITO-9526] - Add the data-index to the timeouts-showcase extended
+* link:https://issues.redhat.com/browse/KOGITO-9317[KOGITO-9317] - Testing CLI + Operator using serverless-workflow-consuming-events-over-http-quarkus
+* link:https://issues.redhat.com/browse/KOGITO-9633[KOGITO-9633] - tzdb.dat (No such file or directory)
+* link:https://issues.redhat.com/browse/KOGITO-9501[KOGITO-9501] - Changing Everit to json-schema-validator
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:27]: appears twice

```suggestion
```

**Review by radtriste — COMMENTED** [review]
I counted those issues missing:
- https://issues.redhat.com/browse/KOGITO-8468
- https://issues.redhat.com/browse/KOGITO-9490
- https://issues.redhat.com/browse/KOGITO-9122

**Comment by ricardozanini** [comment]
> I counted those issues missing:
> 
> * https://issues.redhat.com/browse/KOGITO-8468
> * https://issues.redhat.com/browse/KOGITO-9490
> * https://issues.redhat.com/browse/KOGITO-9122

We need to definitely fix the `fixVersion`. One of them is not even resolved. :(

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 2]
```diff
@@ -3,9 +3,25 @@
...
-// * link:https://issues.redhat.com/browse/KOGITO-XXXX[KOGITO-XXXX] - <description>
-
+* link:https://issues.redhat.com/browse/KOGITO-9584[KOGITO-9584] - Composed JSON Schema does not generate proper swagger documentation
+* link:https://issues.redhat.com/browse/KOGITO-9568[KOGITO-9568] - Remove dmn from kogito-apps and kogito-runtimes if possible
+* link:https://issues.redhat.com/browse/KOGITO-9555[KOGITO-9555] - Add generic endpoint for execution and uploading of process definitions
+* link:https://issues.redhat.com/browse/KOGITO-9415[KOGITO-9415] - Hot reload not working as expected
+* link:https://issues.redhat.com/browse/KOGITO-9393[KOGITO-9393] - [SWF Dev UI] Hot reload custom dashboards
+* link:https://issues.redhat.com/browse/KOGITO-9166[KOGITO-9166] - Gateway Api for management-console-webapp
+* link:https://issues.redhat.com/browse/KOGITO-9244[KOGITO-9244] - Platform.spec.platform.baseImage does not update the Dockerfile base image
+* link:https://issues.redhat.com/browse/KOGITO-9653[KOGITO-9653] - Fix how to handle WaitingForDeployment condition to avoid pod duplication
```
Comment (wmedvede) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:14]:
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-9427[KOGITO-9427] - Improvements in the SW guides for the Timeouts topic
```

**Review by wmedvede — COMMENTED** [review]
@ricardozanini I added a small suggestion to add the improvements related to the Timeouts topic if we have time.

**Review by wmedvede — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

**Comment by ricardozanini** [comment]
@tiagodolphine you can publish the announcement.

**Pull request merged**